packages/flutter/test/widgets/widget_inspector_test_utils.dart

```diff
@@ -24,16 +26,35 @@ class TestWidgetInspectorService extends Object with WidgetInspectorService {
   }
 
   @override
-  void postEvent(String eventKind, Map<Object, Object?> eventData) {
-    getEventsDispatched(eventKind).add(eventData);
+  void postEvent(
+    String eventKind, Map<Object, Object?> eventData,
+    { String stream = 'Extension' }
```

Comment:
opening { should be on previous line, trailing } should be on next line before the )
```
void postEvent(
    String eventKind, Map<Object, Object?> eventData, {
    String stream = 'Extension',
})
```